pom.xml

```diff
@@ -2173,6 +2173,25 @@
         <skip.hudi-spark3.unit.tests>true</skip.hudi-spark3.unit.tests>
         <skipITs>true</skipITs>
       </properties>
+      <build>
+        <plugins>
+          <plugin>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <configuration>
+              <excludes>
```

Comment:
why is this needed? may be I am missing sth obviously.

Reply:
Parquet library version before 1.12.2 does not support column index, e.g., bloom filter. keyed looking up does not work for these < 1.12.2 versions; so we need to exclude these lookup classes.